File: src/Server/RceServer.Front/ClientApp/src/app/shared/job.ts
export enum JobStates {
    Added = 'Added',
    PickedUp = 'PickedUp',
    Updated = 'Updated',
    Success = 'Success',
    Warning = 'Warning',
    Failure = 'Failure',
}

export class Job {
    jobId: string;
    workerId: string;
    name: string;
    payload: any;
    output: any;
    jobState: JobStates;
}

File: src/Server/RceServer.Domain/Models/Job.cs
using Newtonsoft.Json.Linq;

namespace RceServer.Domain.Models
{
	public class Job
	{
		public string JobId { get; set; }
		public string WorkerId { get; set; }
		public string Name { get; set; }
		public JObject Payload { get; set; }
		public JObject Output { get; set; }
		public JobStates JobState { get; set; }
	}
}
